### PR TITLE
Jvenstad/live vespa logs in deployments

### PR DESCRIPTION
**Pull request by jonmv**
@bratseth please review. 

This makes the controller update the Vespa log stored for a deployment job every time a client (console or CLI) asks for logs for that deployment job, i.e., every few seconds. 

The log server gives logs for _at least_ the time window specified, so the time stamp of the last seen entry is used to eliminate duplicates, to micro second precision.

It's possible we need to change the log server to filter entries based on the given time stamp to avoid overloading the network on the controllers, though ... ?

### CODE REVIEW

**Comment by jonmv** [comment]
@olaaun @freva FYI.

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java [hunk 1]
```diff
@@ -30,7 +44,7 @@ public long id() {
         return id;
     }
 
-    public long at() {
+    public Instant at() {
```
Comment (bratseth) [inline controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java]: Are you sure this won't break clients of "controller-api"?

**Review by bratseth — COMMENTED** [review]
Excellent! Just one question

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java (hunk 1 above)
Comment (jonmv) [reply controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java]: Yes.

**Comment by jonmv** [comment]
.

**Pull request merged**